fhir-bucket/src/main/java/com/ibm/fhir/bucket/scanner/DirectoryScanner.java

```diff
@@ -0,0 +1,10 @@
+/*
+ * (C) Copyright IBM Corp. 2020
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package com.ibm.fhir.bucket.scanner;
+
+public class DirectoryScanner {
```

Comment:
I think I missed the intent of this class.  To be implemented?

Reply:
An artifact of my brain as I was thinking how best to scan through the COS tree (wondering if we might need to parallelize scanning of different branches of the tree). Not needed, so I'll remove.